src/graphql/resolvers/monsterResolver.ts

```diff
@@ -1,29 +1,43 @@
-import { levelObjectToArray, resolveChoice, resolveDc, resolveUsage } from './common.js';
+import { levelObjectToArray, resolveChoice, resolveDc } from './common.js';
 
 import ConditionModel from '../../models/condition/index.js';
 import DamageTypeModel from '../../models/damageType/index.js';
 import MonsterModel from '../../models/monster/index.js';
 import ProficiencyModel from '../../models/proficiency/index.js';
 import SpellModel from '../../models/spell/index.js';
 
-const resolveDamage = async damage => {
+import {
+  Monster,
+  ActionDamage,
+  ActionUsage,
+  SpecialAbilityUsage,
+} from '../../models/monster/types';
+
+const resolveUsage = (usage: ActionUsage | SpecialAbilityUsage) => {
+  const resolvedUsage: Record<any, any> = {
+    ...usage,
+    type: usage.type.toUpperCase().replace(/\s+/g, '_'),
+  };
+  if ('rest_types' in usage)
+    resolvedUsage.rest_types = usage.rest_types?.map((rt: string) => rt.toUpperCase());
```

Comment:
You might not need to mark the type of `rt`. Typescript should be able to infer that it's a string.